.circleci/config.yml

```diff
@@ -197,6 +197,22 @@ jobs:
           -X POST https://slack.com/api/chat.postMessage  
 
     - post_steps
+  e2e-preview:
+    docker:
+      - image: mcr.microsoft.com/playwright:v1.41.2-jammy
+    steps:
+      - checkout
+      - run: npm i -D @playwright/test
+      - run: npx playwright install chrome
+      - run: sleep 300
```

Comment:
is there a method to poll? E.g. something like adding a command like `timeout 500 curl -s $E2E_BASE_URL`

Reply:
i looked into it , i do not see a clean, robust way to solve this , suggest to keep the sleep , we could even move it to the end of the preview job as now ci says the preview is ready while in fact the server is starting and the certs are being created.

Reply:
<img width="1142" alt="Screenshot 2024-02-14 at 18 57 27" src="https://github.com/molgenis/molgenis-emx2/assets/47183404/76427d5e-a00f-4c7c-bd18-6ce0269e9a8e">

seems to work, but not really sure its worth the extra code